### ISSUE TITLE
[FEATURE] Pose migration 'passive values' example

### ISSUE THREAD

**Issue by pdkn** [framer/motion#192]
in the Pose Migration docs it could be worth explaining how to migrate the passive values feature of Pose to Motion?

**Comment by InventingWithMonster** [comment]
Yeah that's an excellent point, thanks. You can recreate the passive values with the following code:

```jsx
const x = useMotionValue(0)
const backgroundColor = useTransform(x, [-100, 100], ['#fff', '#000'])

return <motion.div style={{ x, backgroundColor }} />
```

This is a more flexible approach than before as useTransform can attach to any MotionValue from anywhere.

**Comment by InventingWithMonster** [comment]
More info: https://www.framer.com/api/motion/motionvalue

**Comment by pdkn** [comment]
yup, was just working that out - brill ;)
